examples/http3-client.rs

```diff
@@ -46,6 +46,7 @@ Options:
   --no-verify              Don't verify server's certificate.
   --no-grease              Don't send GREASE.
   -H --header HEADER ...   Add a request header.
+  -n --number REQUESTS     Send the given number of identical requests. [default: 1]
```

Comment:
```suggestion
  -n --requests REQUESTS   Send the given number of identical requests [default: 1].
```

`--requests` seems like is a nicer name (same as h2load).